### ISSUE TITLE
Fine-tuning models :  accuracy and loss never converges

### ISSUE THREAD

**Issue by Zappyford** [keras-team/keras#12941]
**System information**  
- Have I written custom code (as opposed to using example directory):  **Yes**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  **Windows 7 Professional**
- TensorFlow backend (yes / no):  **Yes**
- TensorFlow version:  **1.13.1**
- Keras version:  **2.2.4**
- Python version:  **3.6**
- CUDA/cuDNN version:  **10**
- GPU model and memory:  **GeForce GTX 745 2 Go**

**Describe the current behavior**  
Hi !

I'm having a strange behavior during fine-tuning. 

I'm trying to fine-tune Inception on two classes : dogs and cats, with the example from the [documentation](https://keras.io/applications/#fine-tune-inceptionv3-on-a-new-set-of-classes). But when I launch the learning, the accuracy stays around 0.5 and the loss is around 7 (that's a bit huge isn't it ?). 

It seems that there's nothing wrong with the code, so what's happening there ?

I've tried the same code and dataset on another computer and models but the behavior is still the same !

**Describe the expected behavior**  
The model accuracy tends to 1 and the loss to 0.

**Code to reproduce the issue**  
The code : https://gist.github.com/Zappyford/5b6595b8a6596bc14551dbdda5339075

The dataset : https://mega.nz/#!tdYmlAoJ!RMkJjWBKubk7rFZQBngZ52vPF1rXi6FPA6eTQOJYses

**Other info / logs**  

[logs.txt](https://github.com/keras-team/keras/files/3277454/logs.txt)

**Comment by jvishnuvardhan** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue. There are several tutorials in web. [Here](https://gist.github.com/fchollet/7eb39b44eb9e16e59632d25fb3119975) is one close to your work. [Check this also](https://www.learnopencv.com/keras-tutorial-fine-tuning-using-pre-trained-models/).
To ask questions, please see the following resources :
[StackOverflow](http://stackoverflow.com/questions/tagged/keras)
[Keras Slack channel](https://keras-slack-autojoin.herokuapp.com)
[Keras.io support](https://keras.io/#support)
[Gitter](https://gitter.im/Keras-io/Lobby)
Thanks!
If you think we made a mistake, please open another issue and fill all the information asked by the template. Thanks!
